packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -147,8 +147,10 @@ class FlutterIcons {
       UrlIcon('/icons/actions/forceRefresh.png');
   // TODO(dantup): Make a ThemedIcon class to handle this.
   static DevToolsIcon get refresh => isDarkTheme
-      ? const MaterialIcon('refresh', Color.fromARGB(255, 137, 181, 248))
-      : const MaterialIcon('refresh', Color.fromARGB(255, 0, 0, 0));
+      ? MaterialIcon('refresh', const Color.fromARGB(255, 137, 181, 248),
+          codePoint: Icons.refresh.codePoint)
+      : MaterialIcon('refresh', const Color.fromARGB(255, 0, 0, 0),
+          codePoint: Icons.refresh.codePoint);
```

Comment:
can this just be `MaterialIcon('refresh', ThemedColor(Color.fromARGB(255, 137, 181, 248), Color.fromARGB(255, 0, 0, 0)), codePoint: Icons.refresh.codePoint)`?

Reply:
The other way around, but that's definitely an achievable optimization.

Done.